include/nlohmann/detail/input/input_adapters.hpp

```diff
@@ -320,15 +320,18 @@ class input_adapter
                  int>::type = 0>
     input_adapter(IteratorType first, IteratorType last)
     {
+#ifdef DEBUG
```

Comment:
I am sorry, I wrote it wrong in the original issue. This should be `#ifndef NDEBUG`, see https://en.cppreference.com/w/cpp/error/assert.

Reply:
Ah, that's what I thought, should have asked. Brb then!